code/modules/vending/games.dm

```diff
@@ -48,7 +48,8 @@
 		/obj/item/clothing/shoes/wheelys/rollerskates= 3,
 		/obj/item/melee/skateboard/hoverboard = 1,
 		/obj/item/storage/box/tail_pin = 1,
-		/obj/item/toy/intento = 3
+		/obj/item/toy/intento = 3,
+		/obj/item/toy/captainsaid = 1
```

Comment:
```suggestion
		/obj/item/toy/captainsaid = 1,
```

Reply:
Hm? What for?

Reply:
A comma at the end of a list is free and makes it easier to add to the list in the future

Reply:
Sure then